src/plays/tenzies-game/Readme.md

```diff
@@ -0,0 +1,27 @@
+# Tenzies Game
+
+Tenzies Game using React.Js
```

Comment:
Can you please explain about the game a bit more here?

Reply:
Tenzies is a dice roll game where you have to roll the dice until all dice are the same. You can click each die to freeze it at its current value, then you can roll other dice to match the same number on the frozen dice.